docs/cert-compat.md

```diff
@@ -30,7 +30,7 @@ You may want to visit [this particular community forum discussion](https://commu
 * Cyanogen > v10
 * Jolla Sailfish OS > v1.1.2.16
 * Kindle > v3.4.1
-* Java >= JDK 8u101
+* Java >= JDK 7u111, 8u101
```

Comment:
Let's split this into two lines:

- Java 7 >= 7u111
- Java 8 >= 8u101

Also, is this specific to JDK, or does it include JRE? If the former, add back in JDK to my comment; otherwise, leave it out.